src/phases/execution/ClusteredCollection.yml

```diff
@@ -1,6 +1,6 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/server-execution"
-Description: >
+Description: |
```

Comment:
folded block scalars (>) v literal block scalars (|) are going to get tiresome and ugly really quickly ... do we have any way of enforcing the literal block scalars?

Reply:
kk lets leave that for later then.